### PR TITLE
AI Chat (experimental)

### PR DESCRIPTION
**Pull request by jamonholmgren**
# Description

This adds an experimental AI Chat feature to the app that allows the user to ask questions about the conference and surrounding area. It references a (closed source) Next app that secures the API keys and imposes rate limits and other anti-abuse measures.

# Graphics

| iOS            | Android            |
| -------------- | ------------------ |
| CleanShot 2023-04-18 at 16 12 56@2x | CleanShot 2023-04-19 at 15 54 55@2x |

### CODE REVIEW

**Comment by jamonholmgren** [comment]
Later we'll want to add a better icon and I need to do some Android testing and a few other minor changes.

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Coming soon! This will be the home for the app for [Chain React 2023](https://cr
 ## Pre-reqs
 
 1. `brew install fastlane`
-2. `yarn global add eas-cli` or `npm install -g eas-cli` (v3.1.1 is the latest as of this edit)
+2. `npm install -g eas-cli` (v3.1.1 is the latest as of this edit)
```
Comment (mazenchami) [inline README.md:10]: can we put back the `yarn` command? it works for me

**Review by mazenchami — COMMENTED** [review]
Just one README comment. Also, please let me know when you've had the chance to test Android before we get this pushed out to all the users

**Comment by jamonholmgren** [comment]
Android works! 
CleanShot 2023-04-19 at 15 54 55@2x

File: app/navigators/TabNavigator.tsx [hunk 2]
```diff
@@ -61,6 +62,12 @@ const screens: Screen[] = [
     txLabel: "tabNavigator.infoTab",
     icon: "info",
   },
+  {
+    name: "Chat",
+    component: ChatScreen,
+    txLabel: "tabNavigator.chatTab",
+    icon: "ladybug",
```
Comment (morganick) [inline app/navigators/TabNavigator.tsx:69]: Is it a feature or a bug? 🤣 I'll see myself out. 🚶

**Pull request merged**